internal/pkg/print/print.go

```diff
@@ -85,3 +91,27 @@ func (p *Printer) Error(msg string) {
 func (p *Printer) OutOrStdout() io.Writer {
 	return p.Cmd.OutOrStdout()
 }
+
+// Prompts the user for confirmation.
+//
+// Returns nil only if the user (explicitly) answers positive.
+// Returns ErrAborted if the user answers negative.
+func (p *Printer) PromptForConfirmation(cmd *cobra.Command, prompt string) error {
+	question := fmt.Sprintf("%s [y/N] ", prompt)
+	reader := bufio.NewReader(cmd.InOrStdin())
+	for i := 0; i < 3; i++ {
+		cmd.PrintErr(question)
```

Comment:
This one still calls the command's `PrintErr` directly, as it does not depend on the verbosity level, do we want to keep it?

Reply:
I don't think I understood, what do we still want to keep? 
I think its ok that the confirmation prompt does not depend on verbosity, since we would want to have it at every verbosity level right?

Reply:
Yes, the prompts for confirmation don't depend on the verbosity level
We should call `p.cmd.PrintErr` here

Reply:
I was just wondering if we could create a new method in the `print` package that would not depend on the verbosity level and would always run `p.Cmd.PrintErr`

But I think this would be the only use case for it now so I agree we should keep it like this